hack/bump-version.sh

```diff
@@ -29,10 +29,8 @@ if [ -z "$new_version" ]; then
 fi
 
 sed -i "s/$old_version/$new_version/g" cmd/kogito/version/version.go README.md version/version.go deploy/operator.yaml deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml hack/go-build.sh hack/go-vet.sh .osdk-scorecard.yaml
-operator-sdk generate csv --csv-version "$new_version" --from-version "$old_version"  --update-crds --operator-name kogito-operator
+operator-sdk generate csv --apis-dir ./pkg/apis/apps/v1alpha1 --verbose --csv-version "$new_version" --from-version "$old_version"  --update-crds --operator-name kogito-operator
 
-# rewrite test default config, all other configuration into the file will be overridden
-test_config_file="test/.default_config"
```

Comment:
Why is this code removed?

Reply:
because was updating the config file. I'm assuming we don't need that anymore since it's pointing to `latest`. Should we update it every time? If so, I think we could review this approach.

Reply:
ok, thanks
I guess we can remove it

Reply:
This code is important if we use release branches ...
anyway updated in for the future in https://github.com/kiegroup/kogito-cloud-operator/pull/420/files#diff-7a9d206eb5b9303a3c9c293020c798daR47-R50

Reply:
Ok, but aren't you guys changing this configuration all the time? That's what I meant. Should I put it back or not?

Reply:
the `test/.default_config` needs to be there as it is used by the PR checks for example.
it allows also to not set everything each time.
For a release and/or a release branch, we need to change to the correct versions so we are sure the default config is pointing to the right images.

Reply:
Ok, I'll put it back. The `bump-script` changed it to 0.13.0, was latest. Should I keep it as 0.13.0 or latest? Is this the final solution? I mean, we should update this config file every time?

Reply:
if you are on `master`, it should set to `latest` automatically: https://github.com/kiegroup/kogito-cloud-operator/pull/454/files#diff-7a9d206eb5b9303a3c9c293020c798daL39

Reply:
ah but you are not ... that is why XD
need to update the script accordingly then. will do that.
`latest` should be set, also some parameters are missing there. I thought I modified the script already ...
will PR today for the update

Reply:
https://issues.redhat.com/browse/KOGITO-2795

Reply:
that's why I removed these lines. The test config is correctly set to latest. So I believe I have to do any thing here?

Reply:
let it like the default_config is and do not remove the lines. I will update the script.